app-text/simplenote-bin/simplenote-bin-2.22.1.ebuild

```
@@ -0,0 +1,59 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit unpacker xdg
+
+DESCRIPTION="The simplest way to keep notes"
+HOMEPAGE="https://simplenote.com"
+SRC_URI="
+	amd64? (
+		https://github.com/Automattic/simplenote-electron/releases/download/v${PV}/Simplenote-linux-${PV}-amd64.deb\
+			-> simplenote-${PV}-amd64.deb
+	)
+	arm64? (
+		https://github.com/Automattic/simplenote-electron/releases/download/v${PV}/Simplenote-linux-${PV}-arm64.deb\
+			-> simplenote-${PV}-arm64.deb
+	)
+"
+
+S=${WORKDIR}
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~arm64"
+IUSE="doc"
+RESTRICT="bindist mirror"
```

```suggestion
S="${WORKDIR}"

LICENSE="GPL-2"
SLOT="0"
KEYWORDS="-* ~amd64 ~arm64"

IUSE="doc"
RESTRICT="bindist mirror"
```
WORKDIR needs quoting, LICENSE list is probably incomplete since it is an electron app.